### PR TITLE
Migration of test_setting_volume_option_with_more_than_4096_characters

### PR DESCRIPTION
**Pull request by nishith-vihar**
Migration of [test_setting_volume_option_with_more_than_4096_characters.py](https://github.com/gluster/glusto-tests/blob/master/tests/functional/glusterd/test_setting_volume_option_with_more_than_4096_characters.py) test case

Updates: #292
Fixes: #353
Signed-off-by: Nishith Vihar Sakinala nsakinal@redhat.com

### CODE REVIEW

File: tests/functional/glusterd/test_setting_volume_option_with_more_than_4096_characters.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License along
+  with this program; if not, write to the Free Software Foundation, Inc.,
+  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+"""
```
Comment (nik-redhat) [inline tests/functional/glusterd/test_setting_volume_option_with_more_than_4096_characters.py:17]: nit: description

File: tests/functional/glusterd/test_setting_volume_option_with_more_than_4096_characters.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        ip_list = ip_list + ",192.168.123.7"
+        self.options = {"auth.allow": ip_list}
+        redant.logger.info("Setting auth.allow with string of length "
+                           f"{len(ip_list)} for {self.vol_name}")
+        try:
+            redant.set_volume_options(self.vol_name, self.options,
+                                      self.server_list[0])
+        except Exception:
+            redant.logger.info("EXPECTED: setting volume option with more than"
+                               " 4096 characters failed")
```
Comment (nik-redhat) [inline tests/functional/glusterd/test_setting_volume_option_with_more_than_4096_characters.py:66]: It is expected to pass, so update the logic

Comment (nishith-vihar) [reply tests/functional/glusterd/test_setting_volume_option_with_more_than_4096_characters.py:66]: No. It is expected that greater than 4096 will fail. Lesser than 4096 should pass

File: tests/functional/glusterd/test_setting_volume_option_with_more_than_4096_characters.py (hunk 1 above)
Comment (nishith-vihar) [reply tests/functional/glusterd/test_setting_volume_option_with_more_than_4096_characters.py:17]: Ack

File: tests/functional/glusterd/test_setting_volume_option_with_more_than_4096_characters.py (hunk 2 above)
Comment (nik-redhat) [reply tests/functional/glusterd/test_setting_volume_option_with_more_than_4096_characters.py:70]: Where is that written?

**Comment by nik-redhat** [comment]
@srijan-sivakumar this TC fails with timed out in VM env, but as discussed earlier it works for glusto env. Not, sure if that's a code-related issue or env, you can check it in the CI (or else mark it as excluded for the time bn\eing and we can check later).

**Comment by srijan-sivakumar** [comment]
> @srijan-sivakumar this TC fails with timed out in VM env, but as discussed earlier it works for glusto env. Not, sure if that's a code-related issue or env, you can check it in the CI (or else mark it as excluded for the time bn\eing and we can check later).

Cool!

**Review by srijan-sivakumar — APPROVED** [review]

**Pull request merged**